docs-2.0/2.quick-start/2.deploy-nebula-graph-with-docker-compose.md

```diff
@@ -64,15 +64,46 @@ If you are reading this topic with the questions listed below, click them to jum
 
 4. Connect to Nebula Graph.
 
+    1. Start a Nebula Graph service in a Docker container
     ```bash
-    # Start a Nebula Graph service in a Docker container
     $ docker run --rm -ti --network nebula-docker-compose_nebula-net --entrypoint=/bin/sh vesoft/nebula-console:v2-nightly
     ```
+    
+    > **Note**: Your local network  (nebula-docker-compose_nebula-net) may be different from the example above. Use the following command.
+    
+    ```bash
+    $ docker network  ls
+    NETWORK ID          NAME                             DRIVER              SCOPE
+    a74c312b1d16        bridge                           bridge              local
+    dbfa82505f0e        host                             host                local
+    ed55ccf356ae        nebuladockercompose_nebula-net   bridge              local
+    93ba48b4b288        none                             null                local
+
+    ```
 
+    2. Connect to Nebula Graph with Nebula Console
     ```bash
-    # Connect to Nebula Graph with Nebula Console
-    docker> nebula-console -u <user> -p <password> --address=graphd --port=9669
+    docker> nebula-console -u user -p password --address=graphd --port=9669
     ```
+    > **Note**: By default, the authentication is off. Check [TODO(doc)] to turn-on.
+
+    3. Check the Nebula-storaged status, run the `SHOW HOSTS` statement.
+    ```bash
+    nebula> SHOW HOSTS;
+    +-------------+------+----------+--------------+----------------------+------------------------+
+    | Host        | Port | Status   | Leader count | Leader distribution  | Partition distribution |
+    +-------------+------+----------+--------------+----------------------+------------------------+
+    | "storaged0" | 9779 | "ONLINE" | 0            | "No valid partition" | "No valid partition"   |
+    +-------------+------+----------+--------------+----------------------+------------------------+
+    | "storaged1" | 9779 | "ONLINE" | 0            | "No valid partition" | "No valid partition"   |
+    +-------------+------+----------+--------------+----------------------+------------------------+
+    | "storaged2" | 9779 | "ONLINE" | 0            | "No valid partition" | "No valid partition"   |
+    +-------------+------+----------+--------------+----------------------+------------------------+
+    | "Total"     |      |          | 0            |                      |                        |
+    +-------------+------+----------+--------------+----------------------+------------------------+
+    ```
+
+5. Type `exit` twice to switch back to your terminal (shell). You can run Step 4 to login Nebula Graph again.
```

Comment:
Use run instead of type because "run" is the combination of "type" and "press enter".